code/modules/mob/living/silicon/robot/robot_defense.dm

```diff
@@ -1,5 +1,12 @@
 /mob/living/silicon/robot/attackby(obj/item/I, mob/living/user)
 	if(hat_offset != INFINITY && user.a_intent == INTENT_HELP && is_type_in_typecache(I, equippable_hats))
+		if (istype(I, /obj/item/clothing/mask/bandana))
+			if(I.flags_inv > 0)
```

Comment:
what is this check for?

Reply:
The flags_inv variable is zero if the bandana is in hat mode, and something like 168 if it's in mask mode. I was having trouble getting a more appropriate variable to not throw "undefined variable" errors, but this one compiled and worked fine.

Reply:
please use the NONE define
`#define NONE 0`

its' equal to zero but should be used when a var is a bitflag to indicate it is indeed a bandana